.node-version

```diff
@@ -1 +0,0 @@
-20.5.1
```

Comment:
We can't add comments for .node-version https://github.com/shadowspawn/node-version-usage/tree/fb834f2a577cd868c47994563c2799ecc856aedc

Actually rtx crashed